docs/reference/generated/kube-proxy.md

```diff
@@ -80,7 +80,7 @@ TaintNodesByCondition=true|false (ALPHA - default=false)
       --kubeconfig string                            Path to kubeconfig file with authorization information (the master location is set by the master flag).
       --masquerade-all                               If using the pure iptables proxy, SNAT all traffic sent via Service cluster IPs (this not commonly needed)
       --master string                                The address of the Kubernetes API server (overrides any value in kubeconfig)
-      --metrics-bind-address ip                      The IP address and port for the metrics server to serve on (set to 0.0.0.0 for all interfaces) (default 127.0.0.1:10249)
+      --metrics-bind-address ip                      The IP address and port for the metrics server to serve on (set to 0.0.0.0 for all IPv4 interfaces and :: for all IPv6 interfaces) (default 127.0.0.1:10249)
```

Comment:
should be fixed here: https://github.com/kubernetes/kubernetes/blob/master/cmd/kube-proxy/app/server.go#L132